src/Indicator.vala

```diff
@@ -152,6 +152,9 @@ public class DateTime.Indicator : Wingpanel.Indicator {
     }
 
     public override void opened () {
+        Widgets.CalendarModel.get_default ().month_start = Util.get_start_of_month ();
```

Comment:
If this has no effect other than refreshing the events, then I think it would be good to have a comment to that effect above these two lines.

Reply:
Will do.